.eslintrc.js

```diff
@@ -24,5 +24,6 @@ module.exports = {
     'react/destructuring-assignment': 'off',
     'react/state-in-constructor': 'off',
     'react/no-access-state-in-setstate': 'off',
+    'import/no-named-as-default': 0,
```

Comment:
Why did you do this? Where was the issue with this rule?

Reply:
eslint was giving an error in the commit-precheck. Hence I added this line. I will remove it and try running again.